content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -275,7 +275,7 @@ tolerations to all daemons, to prevent DaemonSets from breaking.
 
   * `node.kubernetes.io/memory-pressure`
   * `node.kubernetes.io/disk-pressure`
-  * `node.kubernetes.io/out-of-disk` (*only for critical pods*)
+  * `node.kubernetes.io/pid-pressure` (1.14 or later)
   * `node.kubernetes.io/unschedulable` (1.10 or later)
```

Comment:
I added the "1.14 or later" parenthetical for the `node.kubernetes.io/pid-pressure` taint to be consistent the "1.10 or later" parenthetical for the `node.kubernetes.io/unschedulable` taint.  Let me know if it would make more sense to delete both parentheticals as both versions are years old by now.

Reply:
It probably does make sense to delete both of those.
I'll approve this anyway as _this_ PR is making the website better.